src/main/scala/org/opennetworkinsight/FlowPreLDA.scala

```diff
@@ -171,8 +173,22 @@ object FlowPreLDA {
           }
 
           logger.info("Trying to read file:  " + config.inputPath)
-          val rawdata: RDD[String] = sc.textFile(config.inputPath)
-          val datanoheader: RDD[String] = FlowWordCreation.removeHeader(rawdata)
+          val rawdata: RDD[String] = {
```

Comment:
this is the same code as is in the FlowPostLDA parquet loader... maybe we should have a separate object or class with the  flowData parquet reader that contains this code